build/common.sh

```diff
@@ -1179,7 +1189,26 @@ function kube::release::gcs::verify_release_gt() {
       greater=false
     elif [[ "${version_minor}" -gt "${gcs_version_minor}" ]]; then
       : # fall out
-    elif [[ "${version_patch}" -le "${gcs_version_patch}" ]]; then
+    elif [[ "${version_patch}" -lt "${gcs_version_patch}" ]]; then
```

Comment:
nit: I would much prefer a strictly nested if, with explicit handling of equals, e.g. something along the lines of (in pseudo-code):

```
if major > gcs_major return "greater"
else if major == gcs_major
    if minor > gcs_minor return "greater"
    else if minor == gcs_minor
        return "equal"
    endif
endif
return "less"


```